src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NFloat.cs

```diff
@@ -13,65 +17,691 @@
 
 namespace System.Runtime.InteropServices
 {
-    /// <summary>
-    /// <see cref="NFloat"/> is an immutable value type that represents a floating type that has the same size
-    /// as the native integer size.
-    /// It is meant to be used as an exchange type at the managed/unmanaged boundary to accurately represent
-    /// in managed code unmanaged APIs that use a type alias for C or C++'s <c>float</c> on 32-bit platforms
-    /// or <c>double</c> on 64-bit platforms, such as the CGFloat type in libraries provided by Apple.
-    /// </summary>
+    /// <summary>Defines an immutable value type that represents a floating type that has the same size as the native integer size.</summary>
+    /// <remarks>It is meant to be used as an exchange type at the managed/unmanaged boundary to accurately represent in managed code unmanaged APIs that use a type alias for C or C++'s <c>float</c> on 32-bit platforms or <c>double</c> on 64-bit platforms, such as the CGFloat type in libraries provided by Apple.</remarks>
     [Intrinsic]
-    public readonly struct NFloat : IEquatable<NFloat>
+    public readonly struct NFloat
+        : IComparable,
+          IComparable<NFloat>,
+          IEquatable<NFloat>,
```

Comment:
What about all the static numerical interfaces?  Is that separate?

Reply:
Yes, going to cover that when everything else is fixed up to minimize overall churn.